### PR TITLE
Bug/73678 npm ls fails

### PR DESCRIPTION
**Pull request by jjghali**

### CODE REVIEW

File: package-lock.json
Comment (orck-adrouin) [inline package-lock.json:1]: which node version was used to generate this file?

**Review by orck-adrouin — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (jjghali) [reply package-lock.json:1]: 15.14 was used since it was the same version used in the pipeline

Comment (orck-adrouin) [reply package-lock.json:1]: Can you delete the file and generate a new package-lock.json file using node 18 ? We'll migrate our reacts apps (orc-scripts, orc-shared, oms, analytics, settings) to node18 unless @joclefrancois disagree.

Comment (jjghali) [reply package-lock.json:1]: It is done. Let me know if there are any issues

File: .npmrc [hunk 1]
```diff
@@ -1,3 +1,4 @@
 registry=https://registry.npmjs.org/
 package-lock=false
```
Comment (orck-adrouin) [inline .npmrc]: can you remove the `package-lock=false` line?

**Pull request closed without merging**